mlflow/server/handlers.py

```diff
@@ -446,7 +446,12 @@ def wrapper(*args, **kwargs):
             response.set_data(e.serialize_as_json())
             response.status_code = e.get_http_status_code()
             return response
-
+        except Exception as e:
+            response = Response(mimetype="application/json")
+            e = MlflowException(message="Mlflow server internal error: " + repr(e))
+            response.set_data(e.serialize_as_json())
+            response.status_code = e.get_http_status_code()
+            return response
```

Comment:
Are we sure that the UI will display data, even if the response code does not flag success? (non `2xx` codes)

Reply:
I am not sure how you guys plan on dealing with the "already stored data" issue, perhaps it's a good idea to just cast the timestamp as int and avoid handling this exception entirely?

Reply:
> I am not sure how you guys plan on dealing with the "already stored data" issue, perhaps it's a good idea to just cast the timestamp as int and avoid handling this exception entirely?

This might be a good idea rather than raise error. What do you think @harupy ?

One issue is, your example you use the `time.time()` as timestamp which unit is "second" but the timestamp require unit to be "millisecond", for stored data, we have no way to check whether it use correct unit.

Reply:
Casting the already stored data to int sounds good to me.

Reply:
I think it's also proper to check for type before storing the actual values as well.

Reply:
Agreed, we should do both.